src/sentry/snuba/metrics/fields/snql.py

```diff
@@ -3,26 +3,41 @@
 from sentry.sentry_metrics.utils import resolve_weak
 
 
+def _aggregation_on_session_status_func_factory(aggregate):
+    def _snql_on_session_status_factory(session_status, metric_ids, alias=None):
```

Comment:
to double-check, those metric IDs also end up in the where-clause, right?

Reply:
yes -> https://github.com/getsentry/sentry/blob/b52d8e5fa16670e5d4b071ca72457e187ed7eeeb/src/sentry/snuba/metrics/query_builder.py#L436-L442

Reply:
Good point tho